### DISCUSSION TITLE
debian12 1pctl restart報錯

### DISCUSSION THREAD

**Discussion by fxxkmkd** [1Panel-dev/1Panel#5454] [Q&A]
### 1Panel 版本

v1.10.10-lts  debian12

### 问题描述

安裝後 使用1pctl restart 會提示Jun 13 22:52:48 debian systemd[1]: Started 1panel.service - 1Panel, a modern op>
lines 1-11/11 (END)

### 重现步骤

已經重裝了幾次系統 都提示這個問題
![x](https://github.com/1Panel-dev/1Panel/assets/66674379/8e76b5ad-29ac-4fe8-bc10-d19a12f53f4d)

### 附加信息

[1Panel Log]:  
[1Panel Log]: 为了您的服务器安全，在您离开此界面后您将无法再看到您的密码，请务必牢记您的密码。 
[1Panel Log]:  
[1Panel Log]: ================================================================ 
root@debian:~# 1pctl restart
● 1panel.service - 1Panel, a modern open source linux panel
     Loaded: loaded (/etc/systemd/system/1panel.service; enabled; preset: enabl>
     Active: active (running) since Thu 2024-06-13 22:52:48 EDT; 12ms ago
   Main PID: 16652 (1panel)
      Tasks: 3 (limit: 2307)
     Memory: 4.6M
        CPU: 6ms
     CGroup: /system.slice/1panel.service
             └─16652 /usr/bin/1panel

Jun 13 22:52:48 debian systemd[1]: Started 1panel.service - 1Panel, a modern op>
lines 1-11/11 (END)

**Comment by wanghe-fit2cloud** [comment]
手动执行一下这两条命令：
1、`systemctl restart 1panel.service` 重启
2、`systemctl status 1panel.service` 查看状态

**Reply by openoo** [reply]
确实存在该问题

**Reply by openoo** [reply]
![74547d408e297ca8f6281953f31d397](https://github.com/user-attachments/assets/33964031-964a-47d1-96f7-0cf7c798fea2)
1、systemctl restart 1panel.service 重启
2、systemctl status 1panel.service 查看状态

执行后依然报错

**Comment by RayWangQvQ** [comment]
今天也遇到了，发现是使用香港的节点，访问1panel官网跳`https://1panel.hk/`了，脚本也是用的：

```
curl -sSL https://resource.1panel.hk/quick_start.sh -o quick_start.sh && bash quick_start.sh
```

这个脚本运行后会卡上面的异常，systemctl 重启也没用。

使用1pctl uninstall之后，删除本地文件和脚本，重新用cn的脚本跑，成功了。

建议检查下hk的脚本，是否有bug。
